### DISCUSSION TITLE
How to make an installation package in tB?

### DISCUSSION THREAD

**Discussion by haplox** [twinbasic/twinbasic#1364] [General]
Hello all,

I have been searching without result, so my question is if it is possible to make an installationa package throough tB, as in VB6, for example to include an access database.

**Comment by saturday220514** [comment]
Not answering your question but I think most vb6 users would use inno to make an installer. 
At least that's what I've done for almost all the time I've used vb6, you could use that, it's free.

**Comment by haplox** [comment]
Thanks @saturday220514 , I know INNO, but I expected to have some kind of installatanion maker, as in VB6... I suppouse that INNO is the solution, as there are not official answer to this post

**Comment by haplox** [comment]
What I need to konw is if is neccessary to include some references, DLLs or any other files in order to run the built EXE in any PC

**Comment by saturday220514** [comment]
Hi haplox.
On this page: https://twinbasic.com/preview.html
it says this:

_Generated EXE and DLL files tend to be very small with twinBASIC. Please note that generated EXEs and DLLs do not require any runtime or redistributable components! (unless you reference third-party components yourself)._

**Comment by haplox** [comment]
Thanks @saturday220514 , I ave had not time to read all the documents.

Its clear for me now
